### DISCUSSION TITLE
Android Phone Folder

### DISCUSSION THREAD

**Discussion by Davel95** [WinMerge/winmerge#2409] [Q&A] [answered]
I am trying to compare the "Camera" folder in an Android phone (S23) with a folder on my Windows 10 laptop.  Using WinMerge 2,16.42.1-x64.  WinMerge does not seem to be able to 'see' the phone.  Is there any workaround?

**Comment by sdottaka** [comment, marked as answer]
Unfortunately, WinMerge cannot directly compare the contents of files on an Android phone. However, it seems that Beyond Compare can do this.
To compare files with WinMerge, you will need software to mount your Android as a drive, as shown below.

https://superuser.com/questions/1157661/how-to-mount-an-android-smartphone-as-a-drive-in-windows
https://www.mtpdrive.com/index.html
https://www.guidingtech.com/39472/map-android-drive-windows/

**Reply by Davel95** [reply]
Thanks, It looks like Beyond Compare will do the job for me.
